pkg/utils/rebaseTodo.go

```diff
@@ -32,3 +38,88 @@ func WriteRebaseTodoFile(fileName string, todos []todo.Todo) error {
 	}
 	return err
 }
+
+func MoveTodoDown(fileName string, sha string, action todo.TodoCommand) error {
+	todos, err := ReadRebaseTodoFile(fileName)
+	if err != nil {
+		return err
+	}
+
+	for i := range todos {
+		t := &todos[i]
```

Comment:
Could we instead go `for i, t := range todos {`?

Reply:
Erm, yes. 😅  Also squashed into e9999f3aa8f5.